Front-end/glaucoma-diagnosis-system/src/views/Reception.vue

```diff
@@ -0,0 +1,99 @@
+                                    <option>Oromia</option>
+                                    <option>Sidama</option>
+                                    <option>Somali</option>
+                                    <option>Southern Nation and Nationalities and People</option>
+                                    <option>Tigray</option>
+                                </select>
+                               
+                            </div>
+                            <div class="w-1/2 mr-1">
+                                <label  class="block text-grey-darker background-white text-sm font-bold mb-2" for="sub_city">Sub City</label>
+                                <select class="appearance-none border rounded w-full py-2 px-3 text-grey-darker" id="sub_city" placeholder="">
+                                    <option>Addis Abaa</option>
+                                    <option>Afar</option>
+                                    <option>Amhara</option>
+                                    <option>Benishangul-Gumuz</option>
+                                    <option>Dire Dawa</option>
+                                    <option>Gambela</option>
+                                    <option>Harari</option>
+                                    <option>Oromia</option>
+                                    <option>Sidama</option>
+                                    <option>Somali</option>
+                                    <option>Southern Nation and Nationalities and People</option>
+                                    <option>Tigray</option>
+                                </select>
+                               
+                            </div>
+
+                        </div>
+                        <div class="flex mb-4">
+                            <div class="w-1/2 mr-1">
+                                <label class="block text-grey-darker text-sm font-bold mb-2" for="phone_nubmer">Phone Number</label>
+                                <input class="appearance-none border rounded w-full py-2 px-3 text-grey-darker" id="phone_number" type="text" placeholder="Phone Number">
+                            </div>
+                            <div class="w-1/2 mr-1">
+                                <label  class="block text-grey-darker text-sm font-bold mb-2" for="gender">Gender</label>
+                                <select class="appearance-none border rounded w-full py-2 px-3 text-grey-darker" id="gender" placeholder="">
+                                    <option>Male</option>
+                                    <option>Female</option>
+                                </select>
+                            </div>
+                        </div>
+
+                        <div class="relative text-gray-700">
+                            <button class="bg-blue-500 text-white font-bold py-2 px-4 border border-blue-700 rounded">
+                                Add to Queue
```

Comment:
go to a hospital and ask them about the queue? it is better if u said "register" or something like that "add to queue" seems it is a more techincal word.